streams/src/main/java/org/apache/kafka/streams/kstream/internals/NamedInternal.java

```diff
@@ -67,6 +67,10 @@ String suffixWithOrElseGet(final String suffix, final String other) {
     }
 
     String suffixWithOrElseGet(final String suffix, final InternalNameProvider provider, final String prefix) {
+        return suffixWithOrElseGet(suffix, provider, prefix, false);
```

Comment:
Were these changes put in only for working with `suppress`? If so we can revert them.

Reply:
This modification has been reverted